SEED-platform/seed
# Recognize empty 270

#### Any background context you want to provide?
2.6.1 was patched with a recognize_empty feature per user request. 
https://github.com/SEED-platform/seed/pull/2135

It was also then brought into 2.7.1.
https://github.com/SEED-platform/seed/pull/2136

#### What's this PR do?
This PR brings those changes into 2.7.0.
Each individual commit was cherry-picked from #2136.
The only caveats are the following:
- [Recognize Empty setting shows in column settings change preview](https://github.com/SEED-platform/seed/pull/2159/commits/86827b688b765d4965ca2caec4405d593fddac75) looks more like the [corresponding commit](https://github.com/SEED-platform/seed/pull/2135/commits/bb4e3bbbc1699c5d9d2ef47f78247ce178797303) from #2135 since the merging.py file in 2.7.0 looked more like the one from 2.6.1.
- [Add translated recognize_empty help text to column setting page](https://github.com/SEED-platform/seed/pull/2159/commits/a19f024f49e41f0befff3f6169ac72f77189381b) places the new help text into the format that's also more like 2.6.1.
- Additionally, [Conditional 0119 migration](https://github.com/SEED-platform/seed/pull/2159/commits/a6a0bc604c83144e82db0b252dc3a83282cf8db2) was cherry-picked from https://github.com/SEED-platform/seed/pull/2154.

#### How should this be manually tested?
From #2136 ,
>Toggle the Recognize Empty column setting. Import records that trigger merges and with Not Available or empty values in a particular field. When the setting is checked, see that empty values (truly blank, missing, or "Not Available") are treated like any other non-empty value during a merge.

#### What are the relevant tickets?
#### Screenshots (if appropriate)
